strawberryfields/api_client.py

```diff
@@ -0,0 +1,101 @@
+from urllib.parse import urljoin
+import requests
+
+
+class APIClient:
```

Comment:
Even though this is a scaffolding PR, docstrings should be included in the classes/methods/functions. Even if it's still vague, it would be useful for the engine to know the general use-case of each class/method/function. The details of the docstring can then be filled in on subsequent PRs.

Reply:
Noted.